clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -156,6 +156,10 @@ Changes in existing checks
   `AllowStringArrays` option, enabling the exclusion of array types with deduced
   length initialized from string literals.
 
+- Improved :doc:`bugprone-too-small-loop-variable
```

Comment:
put this in alphabetical order by a check name.

Reply:
I don't think I understand what you mean here. The way I have written is the standard way of writing docs in this file I think. If not please give me the changes that you want to be implemented in this documentation, I will update it.

Reply:
In short, move this release notes entry to line 119, so release notes entry's could be put in an order.

Reply:
Ah, that's what you meant. Done.